# SetJointAngles.srv
float64 elbow_joint
float64 shoulder_lift_joint
float64 shoulder_pan_joint
float64 wrist_1_joint
float64 wrist_2_joint
float64 wrist_3_joint
---
bool success
